src/track/track_gps_l2cm.c

```diff
@@ -0,0 +1,614 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Adel Mamin <adel.mamin@exafore.com>
+ *          Pasi Miettinen <pasi.miettinen@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include "board.h"
+#include "nap/nap_common.h"
+#include "nap/track_channel.h"
+#include "track.h"
+#include "main.h"
+
+#include "track.h"
+
+#define FEATURE_L2C  /* skip weak attributes for L2C API implementation */
```

Comment:
Maybe make the define more specific so you don't have to worry about the undef: `TRACK_GPS_L2CM_INTERNAL` ?